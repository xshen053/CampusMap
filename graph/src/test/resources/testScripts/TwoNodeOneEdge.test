# Tests adding two nodes, one edge between two nodes

# Create a graph
CreateGraph TNOE_graph

# Add two nodes
AddNode TNOE_graph a
AddNode TNOE_graph b

# Add one edge
AddEdge TNOE_graph a b AB

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNOE_graph
ListChildren TNOE_graph a
ListChildren TNOE_graph b