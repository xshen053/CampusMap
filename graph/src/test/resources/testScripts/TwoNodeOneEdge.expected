# Tests adding two nodes, one edge between two nodes

# Create a graph
created graph TNOE_graph

# Add two nodes
added node a to TNOE_graph
added node b to TNOE_graph

# Add one edge
added edge AB from a to b in TNOE_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNOE_graph contains: a b
the children of a in TNOE_graph are: b(AB)
the children of b in TNOE_graph are: